elements/multicolumn/README.md

```diff
@@ -0,0 +1,54 @@
+Multicolumns are the main elements to create complex layouts. They can have multiple columns (we support max. 5 in our editor) and every columns can have children. The children array can contain every element except the full width container. [Read more](//fullwidth_container/README.md)
+
+The arrays in the “cols” array represent the columns. The minimum number of columns is 2 and it can be grown to 5.
```

Comment:
The minimum number stands, but as I mentioned before, the max is only a limitation in the edm editor